probe-rs/src/probe/cmsisdap/commands/mod.rs

```diff
@@ -55,9 +55,12 @@ pub enum CMSISDAPDevice {
     /// CMSIS-DAP v2 over WinUSB/Bulk. Stores an rusb device handle and out/in EP addresses.
     V2 {
         handle: rusb::DeviceHandle<rusb::Context>,
+        // CMSIS-DAP in/out endpoint addresses and max length.
```

Comment:
This comment does not look right for the type below :)

Reply:
Thanks, I meant for the comment to refer to the next three members but it's not very clear. I've updated the docstring on the enum itself instead, hopefully it's clearer now.